src/borg/archiver.py

```diff
@@ -726,8 +726,8 @@ def do_mount(self, args, repository, manifest, key):
             self.print_error('Loading fuse support failed [ImportError: %s]' % str(e))
             return self.exit_code
 
-        if not os.path.isdir(args.mountpoint) or not os.access(args.mountpoint, os.R_OK | os.W_OK | os.X_OK):
-            self.print_error('%s: Mountpoint must be a writable directory' % args.mountpoint)
```

Comment:
Yes, the security check is in the suid binary of course. But an error message from borg is better than just whatever fusermount decides to output.</bikeshed>